.eslintrc

```diff
@@ -19,41 +19,44 @@
     "sourceType": "module"
   },
```

Comment:
I'm guessing this was mostly some re-arranging?

Reply:
Yes, reordering alphabetically, and change rules from numbers to the corresponding string representation (`off`, `warn`, `error`)